pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +142,36 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+type Plugins struct {
+	QueueSort      []Plugin
```

Comment:
We should add a docstring for all these fields. They will be very similar:

// QueueSort is a list of plugins that should be invoked when sorting pods in the scheduling queue.

// PreFilter is a list of plugins that should be invoked at "PreFilter" extension point of the scheduling framework.
...